models/actions/run.go

```diff
@@ -164,6 +164,54 @@ func updateRepoRunsNumbers(ctx context.Context, repo *repo_model.Repository) err
 	return err
 }
 
+// CancelRunningJobs cancels all running jobs of a run
+func CancelRunningJobs(ctx context.Context, run *ActionRun) error {
```

Comment:
```suggestion
// CancelRunningJobs cancels all running jobs of the runs which share the same repo and ref with the inputted run.
func CancelRunningJobs(ctx context.Context, run *ActionRun) error {
```

However, it's still confuing, maybe this could make it clear:

```suggestion
// CancelRunningJobs cancels all running jobs with the repo and ref
func CancelRunningJobs(ctx context.Context, repoID int64, ref string) error {
```

Reply:
Wait, maybe
```suggestion
// // CancelRunningJobs cancels all running jobs with the specified repo, ref and workflow
func CancelRunningJobs(ctx context.Context, repoID int64, ref, workflowID string) error {
```

A run triggered by `a.yaml` shouldn't cancel ones triggered by `b.yaml`.

Reply:
```suggestion
// CancelRunningJobs cancels all running jobs of the specified repo, ref and workflow
func CancelRunningJobs(ctx context.Context, repoID int64, ref, workflowName string) error {
```
Maybe that would sound better?